src/core/config/loader.ts

```diff
@@ -125,7 +125,7 @@ async function _loadUserConfig(
       });
       if (preset) {
         return {
-          config: preset,
+          config: defu(preset),
```

Comment:
`defu` is not safe as a clone utility. we might need to use krona.

Reply:
haha i think @danielroe would be interested to see that :D . Why isn't it safe ?

I'll change move it to klona thanks ! :heart:

Reply:
becaue defu only creates new object on top level (same effect here is { …preset } + when merging two namespaces. It is not for safe clone.

Reply:
Oh that makes sense 👍👍